### PR TITLE
Person Profile Queries Component

### PR DESCRIPTION
**Pull request by amponce**
## Description

Relates to #VACMS-10261. (or closes?) Person Profile

## Testing done

  PersonProfile with valid data
    ✓ renders PersonProfile component (175 ms)
  PersonProfile with invalid data
    ✓ does not render Thumbnail when media is null (10 ms)
    ✓ does not render description when description is null or empty (18 ms)
    ✓ does not render emailAddress when null or empty (13 ms)
    ✓ does not render phoneNumber when null or empty (12 ms)
    ✓ does not render body when completeBiographyCreate is false (6 ms)
    ✓ does not render photo download option when photoAllowHiresDownload is false (5 ms)
    ✓ does not render bio download option when completeBiography is null (4 ms)

Test Suites: 1 passed, 1 total
Tests:       8 passed, 8 total

## Screenshots
personProfile

## Storybook
Screen Shot 2022-09-06 at 2 59 27 PM

## QA steps

Code review
Run branch locally and verify person profile page loads with data

### CODE REVIEW

File: src/templates/common/mediaImage/index.tsx [hunk 1]
```diff
@@ -31,17 +29,15 @@ export const MediaImageComponent = (props: MediaImageType) => {
...
-        <>
-          <Image
-            id={image.id}
-            alt={image.alt}
-            src={imagePath.href || image.url}
-            width={imagePath.width || image.width}
-            height={imagePath.height || image.height}
-            className={props?.className}
-          />
-          <span>{image.title}</span>
```
Comment (penny-lischer) [inline src/templates/common/mediaImage/index.tsx:43]: Do we need to keep this logic?

File: src/templates/components/newsStoryTeaser/index.test.tsx [hunk 2]
```diff
@@ -48,7 +38,7 @@ describe('<NewsStoryTeaser> with valid data', () => {
     const { container } = render(<NewsStoryTeaser {...teaserData} />)
     const imgEl = container.querySelectorAll('img')
 
-    expect(imgEl.length).toBe(0) // fix this
+    expect(imgEl.length).toBe(2) // fix this
```
Comment (penny-lischer) [inline src/templates/components/newsStoryTeaser/index.test.tsx:41]: Can this comment be removed?

File: src/templates/common/mediaImage/index.tsx (hunk 1 above)
Comment (amponce) [reply src/templates/common/mediaImage/index.tsx:43]: No, we moved that into the image component directly after finding out there was support for title and tooltips 

```        <Image
          id={image.id}
          alt={image.alt}
          src={imagePath.href}
          width={imagePath.width}
          height={imagePath.height}
          title={image.title}
          className={props?.className}
        />```

File: src/templates/components/newsStoryTeaser/index.test.tsx (hunk 2 above)
Comment (amponce) [reply src/templates/components/newsStoryTeaser/index.test.tsx]: Yep, fixed!

**Pull request merged**